pkg/ccl/streamingccl/streamproducer/event_stream.go

```diff
@@ -380,6 +380,7 @@ func (s *eventStream) streamLoop(ctx context.Context, frontier *span.Frontier) e
 				}
 			default:
 				// TODO(yevgeniy): Handle SSTs.
+				// TODO(erikgrinaker): Handle DeleteRange events (MVCC range tombstones).
```

Comment:
@miretskiy @dt Do we need to address these TODOs before stability in order to test streaming replication for 22.1? That is, add handling for `AddSSTable` and MVCC range tombstones in streaming replication.

Reply:
We decided that we will test what we can -- even if some of those ops are not ready yet.
So, these TODOs are fine for now.

Reply:
Ok. We have pending PRs/issues for the necessary APIs to ingest these on the target cluster, and plan to get these in for 22.1 in experimental form. We'll be maturing these APIs during stability and the 22.1 cycle. So if we get the streaming ingestion code in place before stability as well (i.e. simply make the appropriate API calls) then we should be able to do full end-to-end testing of streaming replication sometime during the 22.1 cycle.